app/src/main/java/org/mozilla/fenix/gecko/GeckoProvider.kt

```diff
@@ -57,7 +57,7 @@ object GeckoProvider {
             .telemetryDelegate(GeckoAdapter())
             .contentBlocking(policy.toContentBlockingSetting())
             .debugLogging(Config.channel.isDebug)
-            .aboutConfigEnabled(true)
+            .aboutConfigEnabled(Config.channel.isBeta || Config.channel.isNightlyOrDebug)
```

Comment:
@rocketsroger @Amejia481 We should get rid of the now unused `GeckoProvider` under `/geckoRelease`: https://github.com/mozilla-mobile/fenix/blob/master/app/src/geckoRelease/java/org/mozilla/fenix/engine/GeckoProvider.kt

In facct we don't need a provider class anymore and could just fold this into `Core.kt` directly. Most importantly we also need to verify the two providers are otherwise identical. Can one of you do that as a follow up, please? :)

Reply:
Understood, will do

Reply:
Issue created https://github.com/mozilla-mobile/fenix/issues/20285